rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -493,6 +510,19 @@ def _decode_transaction(
             counterparties=counterparties,
         )
 
+        if monerium_special_handling_event is True:
+            new_events = []  # we create a new list to avoid remove operations and modifying while iterating  # noqa: E501
+            replacements = self.exceptions_mappings.values()
+            for event in events:
+                if (replacement := self.exceptions_mappings.get(event.asset.identifier)) is not None:  # noqa: E501
+                    event.asset = Asset(replacement)
```

Comment:
Since you turn it into an asset shouldn't the mapping contain Assets instead?

Reply:
Didn't want to store the wrapper in memory for a case that might not ever exist for the user but it might not be that much.